metals/src/main/scala/scala/meta/internal/implementation/ImplementationProvider.scala

```diff
@@ -328,10 +372,10 @@ final class ImplementationProvider(
         loc =>
           // we are not interested in local symbols from outside the workspace
           (loc.symbol.isLocal && loc.file.isEmpty) ||
-          // local symbols ineheritance should only be picked up in the same file
+          // local symbols inheritance should only be picked up in the same file
           (loc.symbol.isLocal && loc.file != currentPath)
       }
-      directImplementations.toSet ++ directImplementations
+      directImplementations ++ directImplementations
```

Comment:
` directImplementations ++ directImplementations` I think we might only need  `directImplementations.toSet` - this actually never made sense :fearful: my bad